lib/configuration/is-dashboard-enabled.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = ({ configuration, options }) => {
+  if (configuration) {
+    if (configuration.console && !configuration.dashboard) return false;
```

Comment:
Does that mean that you can configure both console and dashboard for the same service?

Reply:
By default we turn off dashboard integration when console is on, but we want to allow users to use both, for e.g. migration purposes

Reply:
Thanks for clarification 👍